test/support/token_module/token_module.ex

```diff
@@ -146,4 +121,8 @@ defmodule Guardian.Support.TokenModule do
       {:ok, {old_token, old_claims}, {new_t, new_c}}
     end
   end
+
+  defp send_function_call(call) do
```

Comment:
Where is this used?

Reply:
In this file `test/support/token_module/token_module.ex`